### PR TITLE
Patch messages in SSB model vocabulary

### PR DESCRIPTION
**Pull request by pawbu**

### CODE REVIEW

File: job_executor/model/metadata.py [hunk 1]
```diff
@@ -243,9 +258,9 @@ def validate_patching_fields(
     def validate_represented_variables(self, other: 'Variable'):
         if len(self.represented_variables) != len(other.represented_variables):
             raise PatchingError(
-                'Can not change the number of represented variables '
-                f'from {len(self.represented_variables)} to {len(other.represented_variables)} '
-                'Please check valueDomain.codeList field.'
+                'Can not change the number of code list time periods '
```
Comment (DanielElisenberg) [inline job_executor/model/metadata.py:261]: :100:

File: job_executor/model/metadata.py [hunk 2]
```diff
@@ -188,6 +196,11 @@ def remove_none(cls, values):
             if value is not None
         }
 
+    @staticmethod
+    def data_type_to_ssb_model(data_type: str):
```
Comment (DanielElisenberg) [inline job_executor/model/metadata.py:200]: Any reason we are using this method rather than looking up in the module dictionary directly?

Comment (pawbu) [reply job_executor/model/metadata.py]: No reason :) I fix

**Review by DanielElisenberg — APPROVED** [review]
:100:

**Pull request merged**